Listing '.'...
Listing './bin'...
Listing './include'...
Listing './include/site'...
Listing './include/site/python3.5'...
Listing './include/site/python3.5/zope.proxy'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/RestrictedPython'...
Compiling './lib/python3.5/site-packages/RestrictedPython/Eval.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/Eval.py", line 63
    print 'prepRestrictedCode: %d ms for %s' % (
                                           ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/RestrictedPython/Guards.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/Guards.py", line 101
    raise TypeError, error_msg
                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/RestrictedPython/Limits.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/Limits.py", line 28
    raise AttributeError, 'range() requires 1-3 int arguments'
                        ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/RestrictedPython/RCompile.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/RCompile.py", line 61
    raise SyntaxError, self.rm.errors[0]
                     ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/RestrictedPython/tests'...
Compiling './lib/python3.5/site-packages/RestrictedPython/tests/before_and_after.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/tests/before_and_after.py", line 80
    print "foo"
              ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/RestrictedPython/tests/class.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/tests/class.py", line 13
    raise AssertionError, "expected 13, got %d" % x.get()
                        ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/RestrictedPython/tests/restricted_module.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/tests/restricted_module.py", line 4
    print 'Hello, world!',
                        ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/RestrictedPython/tests/security_in_syntax.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/tests/security_in_syntax.py", line 32
    exec 'q = 1'
               ^
SyntaxError: Missing parentheses in call to 'exec'

Compiling './lib/python3.5/site-packages/RestrictedPython/tests/testRestrictions.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/tests/testRestrictions.py", line 79
    exec code in rmodule
            ^
SyntaxError: Missing parentheses in call to 'exec'

Compiling './lib/python3.5/site-packages/RestrictedPython/tests/unpack.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/tests/unpack.py", line 23
    raise AssertionError, "expected 'unpack list of wrong size'"
                        ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/RestrictedPython/tests/verify.py'...
***   File "./lib/python3.5/site-packages/RestrictedPython/tests/verify.py", line 86
    print i, op.opname, op.arg
          ^
SyntaxError: Missing parentheses in call to 'print'

Listing './lib/python3.5/site-packages/RestrictedPython-3.6.0.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/zope'...
Listing './lib/python3.5/site-packages/zope/component'...
Listing './lib/python3.5/site-packages/zope/component/testfiles'...
Listing './lib/python3.5/site-packages/zope/component/tests'...
Listing './lib/python3.5/site-packages/zope/event'...
Listing './lib/python3.5/site-packages/zope/i18nmessageid'...
Listing './lib/python3.5/site-packages/zope/interface'...
Listing './lib/python3.5/site-packages/zope/interface/common'...
Listing './lib/python3.5/site-packages/zope/interface/common/tests'...
Listing './lib/python3.5/site-packages/zope/interface/tests'...
Listing './lib/python3.5/site-packages/zope/location'...
Listing './lib/python3.5/site-packages/zope/location/tests'...
Listing './lib/python3.5/site-packages/zope/proxy'...
Listing './lib/python3.5/site-packages/zope/proxy/tests'...
Listing './lib/python3.5/site-packages/zope/schema'...
Listing './lib/python3.5/site-packages/zope/schema/tests'...
Listing './lib/python3.5/site-packages/zope/security'...
Listing './lib/python3.5/site-packages/zope/security/examples'...
Listing './lib/python3.5/site-packages/zope/security/tests'...
Listing './lib/python3.5/site-packages/zope/untrustedpython'...
Compiling './lib/python3.5/site-packages/zope/untrustedpython/interpreter.py'...
***   File "./lib/python3.5/site-packages/zope/untrustedpython/interpreter.py", line 23
    exec code in globals, locals
            ^
SyntaxError: Missing parentheses in call to 'exec'

Compiling './lib/python3.5/site-packages/zope/untrustedpython/tests.py'...
***   File "./lib/python3.5/site-packages/zope/untrustedpython/tests.py", line 25
    exec 'x = str(1)' in d
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/zope.component-4.4.1.dist-info'...
Listing './lib/python3.5/site-packages/zope.event-4.3.0.dist-info'...
Listing './lib/python3.5/site-packages/zope.i18nmessageid-4.1.0.dist-info'...
Listing './lib/python3.5/site-packages/zope.interface-4.4.3.dist-info'...
Listing './lib/python3.5/site-packages/zope.location-4.1.0.dist-info'...
Listing './lib/python3.5/site-packages/zope.proxy-4.3.0.egg-info'...
Listing './lib/python3.5/site-packages/zope.schema-4.5.0.dist-info'...
Listing './lib/python3.5/site-packages/zope.security-4.2.2.egg-info'...
Listing './lib/python3.5/site-packages/zope.untrustedpython-4.0.0.egg-info'...
Listing './share'...
Listing './share/python-wheels'...
